box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxMetadataUpdateTask.java

```diff
@@ -0,0 +1,83 @@
+package com.box.androidsdk.content.models;
+
+/**
+ * BoxMetadataUpdateTask instances are used for updating metadata with BoxMetadataUpdateRequest.
+ */
+public class BoxMetadataUpdateTask extends BoxJsonObject {
+
+    /**
+     * Operation to perform (add, replace, remove, test).
+     */
+    public static String OPERATION = "op";
+
+    /**
+     * Path (key) to update.
+     */
+    public static String PATH = "path";
+
+    /**
+     * Value to use (not required for remove operation).
+     */
+    public static String VALUE = "value";
+
+    /**
+     * ENUM that defines all possible operations available to the BoxMetadataUpdateTask class.
+     */
+    public enum BoxMetadataUpdateOperations {
```

Comment:
Since this enum is inside BoxMetadataUpdateTask let's just call it Operations for brevity